npm/cypress-schematic/src/schematics/ng-add/index.spec.ts

```diff
@@ -32,7 +32,7 @@ describe('@cypress/schematic: ng-add', () => {
   })
 
   it('should create cypress files', async () => {
-    const files = ['cypress/integration/spec.ts', 'cypress/plugins/index.ts', 'cypress/support/commands.ts', 'cypress/support/index.ts', 'cypress/tsconfig.json', 'cypress.config.ts']
+    const files = ['cypress/e2e/spec.ts', 'cypress/plugins/index.ts', 'cypress/support/commands.ts', 'cypress/support/index.ts', 'cypress/tsconfig.json', 'cypress.config.ts']
```

Comment:
Shouldn't this be `cypress/e2e/spec.cy.ts`? 
```suggestion
    const files = ['cypress/e2e/spec.cy.ts', 'cypress/plugins/index.ts', 'cypress/support/commands.ts', 'cypress/support/index.ts', 'cypress/tsconfig.json', 'cypress.config.ts']
```

Reply:
Yes, I will change it, good catch. I don't know much about schematics, I will ping the DX team to find out how best to handle this release.

Reply:
https://github.com/cypress-io/cypress/pull/19345/commits/ee2e6e5db426040635b89e992230ebe23e3af129